internal/runner/rspec_test.go

```diff
@@ -1,12 +1,71 @@
 package runner
 
 import (
+	"errors"
 	"os"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"github.com/kballard/go-shellquote"
 )
 
+func TestCommandNameAndArgs_WithInterpolationPlaceholder(t *testing.T) {
+	rspec := Rspec{}
+	testCases := []string{"spec/models/user_spec.rb", "spec/models/billing_spec.rb"}
+	testCommand := "bin/rspec --options {{testExamples}} --format"
+
+	gotName, gotArgs, _ := rspec.commandNameAndArgs(testCases, testCommand)
+
+	wantName := "bin/rspec"
+	wantArgs := []string{"--options", "spec/models/user_spec.rb", "spec/models/billing_spec.rb", "--format"}
+
+	if diff := cmp.Diff(gotName, wantName); diff != "" {
+		t.Errorf("Rspec.commandNameAndArgs() diff (-got +want):\n%s", diff)
+	}
+	if diff := cmp.Diff(gotArgs, wantArgs); diff != "" {
+		t.Errorf("Rspec.commandNameAndArgs() diff (-got +want):\n%s", diff)
+	}
+}
+
+func TestCommandNameAndArgs_WithoutInterpolationPlaceholder(t *testing.T) {
+	rspec := Rspec{}
+	testCases := []string{"spec/models/user_spec.rb", "spec/models/billing_spec.rb"}
+	testCommand := "bin/rspec --options --format"
+
+	gotName, gotArgs, _ := rspec.commandNameAndArgs(testCases, testCommand)
+
+	wantName := "bin/rspec"
+	wantArgs := []string{"--options", "--format", "spec/models/user_spec.rb", "spec/models/billing_spec.rb"}
+
+	if diff := cmp.Diff(gotName, wantName); diff != "" {
+		t.Errorf("Rspec.commandNameAndArgs() diff (-got +want):\n%s", diff)
+	}
+	if diff := cmp.Diff(gotArgs, wantArgs); diff != "" {
+		t.Errorf("Rspec.commandNameAndArgs() diff (-got +want):\n%s", diff)
+	}
+}
+
+func TestCommandNameAndArgs_InvalidTestCommand(t *testing.T) {
+	rspec := Rspec{}
+	testCases := []string{"spec/models/user_spec.rb", "spec/models/billing_spec.rb"}
+	testCommand := "bin/rspec --options ' {{testExamples}}"
+
+	gotName, gotArgs, err := rspec.commandNameAndArgs(testCases, testCommand)
+
+	wantName := ""
+	wantArgs := []string{}
+
+	if diff := cmp.Diff(gotName, wantName); diff != "" {
+		t.Errorf("Rspec.commandNameAndArgs() diff (-got +want):\n%s", diff)
+	}
+	if diff := cmp.Diff(gotArgs, wantArgs); diff != "" {
+		t.Errorf("Rspec.commandNameAndArgs() diff (-got +want):\n%s", diff)
+	}
```

Comment:
If the intention is to disregard the other return values when `err != nil`, then there's not much value in testing that the other return values are empty. But this is fine.